### DISCUSSION TITLE
Development environment installation

### DISCUSSION THREAD

**Discussion by adiityack** [juspay/hyperswitch#4693] [General]
I am interested in contributing to the HyperSwitch project and need some assistance with the initial setup and a small task.
Can anyone please provide me detailed instructions on
**Setting up the development environment for HyperSwitch.**
Any guidance on the prerequisites, dependencies, and configuration steps would be greatly appreciated.

**Comment by SanchithHegde** [comment]
@adiityack Please refer to our [local setup guide](https://github.com/juspay/hyperswitch/blob/main/docs/try_local_system.md) for setting up your development environment for the app server. Depending on your preference, you can either choose to set it up with Docker Compose, or install Rust and other dependencies directly on your system (you can jump to the relevant section from the table of contents).

If you'd like to contribute to the [web client](https://github.com/juspay/hyperswitch-web#%EF%B8%8F-setup-instructions) or the [control center](https://github.com/juspay/hyperswitch-control-center#standard-installation) repositories, please refer to their documentation files.
